.all-contributorsrc

```diff
@@ -232,7 +232,7 @@
       "avatar_url": "https://avatars2.githubusercontent.com/u/8073251?v=4",
       "profile": "https://github.com/xuchaoying",
       "contributions": [
-        "code",
```

Comment:
This was causing all-contributors to crash hard with a useless error message `Cannot read property 'then' of null` I'm going to propose we improve the parsing of the config file, to alert on issues like this

Reply:
added better detection of this in: https://github.com/jfmengels/all-contributors-cli/pull/124/files